tests/robot-tests/tests/snapshots/data_catalogue_snapshot.json

```diff
@@ -1,5 +1,5 @@
 {
-  "num_datasets": "959 data sets",
+  "num_datasets": "941 data sets",
   "themes": [
     {
```

Comment:
Looking at the number of datasets that have gone down from 959 to 941

Reply:
Confirmed now as the Higher Level Learners release 2022/23. That's dropped from 35 to 17 since the last release (2021/22).